lib/controller/show.js

```diff
@@ -0,0 +1,34 @@
+const showInput = require('../cli/input/show');
+const showOutput = require('../cli/output/show');
+const showModel = require('../model/show');
+
+const showValidate = require('./validate/show');
+
+const onShowResults = (showResult) => {
+  showOutput.sendOutput(showResult);
+};
+
+const onShow = async (args, options, logger) => {
+  logger.moduleLog('info', 'Exit', 'Show command invoked.');
+
+  const showOptions = await showInput.getInput(args, options);
+  const validatedOptions = showValidate.validate(showOptions);
+
+  logger.moduleLog('debug', 'Eval', 'Show request for');
```

Comment:
These two log statements again belong to `Show` module, not `Eval` and the statements can be better.

Reply:
Fixing it.